API-Reference/Macros.md

```diff
@@ -76,25 +76,81 @@ with.
 | `KOKKOS_ENABLE_LIBDL`       | Defined if Kokkos links to the dynamic linker (libdl).                                                             |
 | `KOKKOS_ENABLE_LIBQUADMATH` | Defined if Kokkos links to the [GCC Quad-Precision Math Library API](https://gcc.gnu.org/onlinedocs/libquadmath/). |
 
+## Compilers
+
+The following macros can be used to query the integral version values of used compilers.
+
+| Macro | Description |
+| ------| ---------------|
+| `KOKKOS_COMPILER_NVCC` | Defined if NVIDIA compiler is used. |
+| `KOKKOS_COMPILER_GNU`  | Defined if GCC compiler is used. |
+| `KOKKOS_COMPILER_INTEL` | Defined if Intel compiler or Intel LLVM compiler is used. |
```

Comment:
I think `Intel` calls them nowadays `Intel Compiler Classic` and `Intel oneAPI DPC++/C++ Compiler`, see https://www.intel.com/content/www/us/en/develop/documentation/cpp-compiler-developer-guide-and-reference/top.html and https://www.intel.com/content/www/us/en/develop/documentation/oneapi-dpcpp-cpp-compiler-dev-guide-and-reference/top.html.